src/paint.rs

```diff
@@ -260,6 +260,9 @@ impl<'a> Painter<'a> {
                 .iter()
                 .zip_eq(diff_style_sections.iter()),
         ) {
+            if state == &State::HunkHeader && config.color_only {
+                continue;
+            }
```

Comment:
In specific, 
```
@@ -444,6 +444,8 @@ fn handle_hunk_header_line(
```

will be formatted to 

```
fn handle_hunk_header_line(
```

in this part. So I added only when state is hunkheader and color-only is enabled, skip the process.

Reply:
this won't be needed as same as https://github.com/dandavison/delta/pull/323#r490649549

Reply:
Fixed here 8a5fb65